Packs/MajorBreachesInvestigationandResponse/pack_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "Rapid Breach Response",
-    "description": "This content pack helps you to collect, investigate, and remediate incident relates to major breaches.",
+    "description": "This content pack helps you collect, investigate, and remediate incident relate to major breaches.",
```

Comment:
This content pack helps you collect, investigate, and remediate incidents related to major breaches.